src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -18,21 +20,23 @@ import {
   graphql,
   RelayProp,
 } from "react-relay"
+import { TrackingProp } from "react-tracking"
 import { data as sd } from "sharify"
 import { get } from "Utils/get"
 import createLogger from "Utils/logger"
 
 const logger = createLogger("Apps/Auction/Routes/ConfirmBid")
 
-interface BidProps {
+interface ConfirmBidProps {
   artwork: routes_ConfirmBidQueryResponse["artwork"]
   me: routes_ConfirmBidQueryResponse["me"]
   relay: RelayProp
   location: Location
+  tracking: TrackingProp
 }
 
-export const ConfirmBidRoute: React.FC<BidProps> = props => {
-  const { me, artwork } = props
+export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
+  const { me, artwork, tracking } = props
```

Comment:
Better to use the `useTracking` hook rather than pluck from props

Reply:
does this mean removing the tracking prop?

Reply:
Meaning the interface? Don't need it if it's not a prop. The `useTracking` value comes from our context.

Reply:
@damassi I believe (but could be wrong) that Erik and I tried to this more hooks oriented approach when implementing tracking for the Register component (https://github.com/artsy/reaction/pull/2727), but ran into issues with Formik.

@bhoggard I'd timeboxing ~1 hour to trying out this refactoring, because it seems worth it if it doesn't take too long!